### PR TITLE
initial-home-page

### PR DESCRIPTION
**Pull request by iamharnad**
created a structure and added context which is required for the quiz app

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
```
Comment (Mahi24-prog) [inline public/index.html:26]: Remove unnecessary comments

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
```
Comment (Mahi24-prog) [inline public/manifest.json:25]: Remove unnecessary files

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
```
Comment (Mahi24-prog) [inline src/App.css:4]: you can use utility classes for text center.

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
```
Comment (Mahi24-prog) [inline src/App.css:18]: use color variable.

File: src/App.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
```
Comment (Mahi24-prog) [inline src/App.css:25]: use utility classes for flex properties.

File: src/Context/Context.jsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  const [index, setIndex] = useState(0);
+  const [correct, setCorrect] = useState(0);
+  const [error, setError] = useState(false);
+  const [quiz, setQuiz] = useState({
+    amount: 10,
+    category: 'sports',
+    difficulty: 'ease',
+  });
+  const [modal, setModal] = useState(false);
+
```
Comment (Mahi24-prog) [inline src/Context/Context.jsx:20]: You can use useRecucer for multiple states handle.

File: src/index.css [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+===============
+*/
+
+main {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.quiz {
```
Comment (Mahi24-prog) [inline src/index.css:51]: Use utility classes for flex properties

**Review by Mahi24-prog — COMMENTED** [review]
Go through suggestion given.

**Review by rehanmohiuddin — APPROVED** [review]

**Pull request merged**